hbase-server/src/main/java/org/apache/hadoop/hbase/quotas/SpaceQuotaRefresherChore.java

```diff
@@ -74,6 +76,12 @@ public SpaceQuotaRefresherChore(RegionServerSpaceQuotaManager manager, Connectio
   @Override
   protected void chore() {
     try {
+      // check whether Quota table is present or not.
+      if (!quotaTablePresent && !checkQuotaTableExists()) {
+        LOG.info("Quota table not found, skipping quota manager cache refresh.");
+        quotaTablePresent = true;
```

Comment:
Is this right? Setting it to true? Should it be false? The table was 'not found'.

Reply:
ohh yes..I should set it after the if condition...my bad :)

Reply:
Thanks for the review. Made the  changes. :)